README.adoc

```diff
@@ -1167,19 +1167,32 @@ And you'll have to consider the fact that most non-trivial apps share a database
 
 === 3-state Boolean [[three-state-boolean]]
 
-With SQL databases, if a boolean column is not given a default value, it will have three possible values: `true`, `false` and `NULL`.
+With SQL databases, if a boolean column is nullable, it will have three possible values: `true`, `false` and `NULL`.
 Boolean operators https://en.wikipedia.org/wiki/Three-valued_logic[work in unexpected ways] with `NULL`.
 
 For example in SQL queries, `true AND NULL` is `NULL` (not false), `true AND NULL OR false` is `NULL` (not false). This can make SQL queries return unexpected results.
 
-To avoid such situations, boolean columns should always have a default value and a `NOT NULL` constraint.
+To avoid such situations, boolean columns should always have a `NOT NULL` constraint.
+
+Note that when adding a boolean column to an existing table, a default value should be put in place. Otherwise the `NOT NULL` constraint will break for existing rows.
```

Comment:
Or it can be added without the constraint, backfilled, and have the constraint added afterwards.

Reply:
True ! Though adding the default is safer because between the backfill and adding the constraint, new rows might be inserted

Would you like to see a small phrase about the "no constraint, backfill, constraint" procedure ?